data/network/src/main/java/io/filmtime/data/network/NetworkModule.kt

```diff
@@ -24,16 +27,38 @@ object NetworkModule {
     }
   }
 
+  @Provides
+  @Singleton
+  fun providesInterceptor(): Map<Int, @JvmSuppressWildcards Interceptor> {
+    return mapOf(
+      1 to ApiKeyInterceptor()
```

Comment:
I think it's better to provide different interceptor instances by annotation classes